package.json

```diff
@@ -34,12 +38,16 @@
   },
   "devDependencies": {
     "@babel/core": "^7.0.0",
+    "@graphql-codegen/cli": "^1.13.2",
+    "@graphql-codegen/typescript": "^1.13.2",
+    "@graphql-codegen/typescript-operations": "^1.13.2",
     "@types/jest": "^25.1.4",
     "@types/node": "^13.9.2",
     "@types/react": "~16.9.0",
     "@types/react-native": "~0.60.23",
     "@typescript-eslint/eslint-plugin": "^2.25.0",
     "@typescript-eslint/parser": "^2.25.0",
+    "apollo-codegen": "^0.20.2",
```

Comment:
You decided not to use this right? Can we remove?

Reply:
that's right